packages/gatsby/src/utils/__tests__/get-ssl-cert.ts

```diff
@@ -28,9 +28,9 @@ describe(`gets ssl certs`, () => {
     certificateFor.mockClear()
   })
   describe(`Custom SSL certificate`, () => {
-    it.each([[{ certFile: `foo` }], [{ keyFile: `bar` }]])(
+    it.each([{ certFile: `foo` }, { keyFile: `bar` }])(
       `panic if cert and key are not both included`,
-      args => {
+      (args: any): void => {
```

Comment:
Why not cast this explicitly to `IGetSslCertArgs`?

Reply:
fixed